.buildkite/scripts/packaging/package-dra.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
We are missing the `set -ueo pipefail` so that if a step fails the script fails

Reply:
Done